contracts/cw20-merkle-airdrop/src/error.rs

```diff
@@ -32,6 +32,9 @@ pub enum ContractError {
     #[error("Airdrop stage {stage} expired at {expiration}")]
     StageExpired { stage: u8, expiration: Expiration },
 
+    #[error("Airdrop stage {stage} not expired yet")]
```

Comment:
```suggestion
    #[error("Airdrop stage {stage} expires at {expiration}")]
```
You may need to implement `Display` trait.